SU-UComm/stanford-report
# STANFORD-419 | CTA cards block

# READY FOR REVIEW

# Summary
- CTA Cards Block component

# Review By (Date)
- Whenever you need it

# Criticality
- 5

# Review Tasks

## Setup tasks and/or behavior to test

1. See my test page
https://sug-web.matrix.squiz.cloud/report/development/yvonnes-test-page
2. Test different responsive breakpoints
3. Test both light and dark mode
4. Change background to #f4f4f4 (fog light) in the inspector and see that the card still look good - this is simulating the future Campaign page. Below is my test:
![Screenshot 2024-03-25 at 4 31 02 PM](https://github.com/SU-UComm/stanford-report/assets/42749717/956fbe6e-8656-4975-b99d-86ac4c15da14)

5. Add the component on your own test page and see the user experience
6. Look at code

# Associated Issues and/or People
- JIRA ticket(s) - STANFORD-419
